src/main/java/org/dasein/cloud/compute/MachineImage.java

```diff
@@ -80,9 +80,11 @@
         image.type = MachineImageType.VOLUME;
         image.creationTimestamp = 0L;
         image.software = "";
+        image.minimumDiskSizeGb = 10L;
         return image;
     }
 
+    private long               minimumDiskSizeGb;
```

Comment:
All other instances of gigabyte-related fields in core are of type int. Consistency is our friend!